src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -22,8 +22,10 @@ const AVATAR_SIZE = 24;
 const CONTAINER_LEFT_PADDING = 60;
 const CONTAINER_RIGHT_PADDING = 10;
 const AVATAR_LEFT_MARGIN = 4;
-const BUBBLE_LEFT_PADDING = 4;
-const BUBBLE_RIGHT_PADDING = 4;
+const BUBBLE_LEFT_PADDING = 8;
+const BUBBLE_RIGHT_PADDING = 8;
+const BUBBLE_LEFT_PADDING_ATTACHMENT = 4;
+const BUBBLE_RIGHT_PADDING_ATTACHMENT = 4;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The padding for the chat bubbles has been increased. This might affect the layout and appearance of the chat bubbles. Please ensure that this change is intentional and does not negatively impact the user interface.

```diff
- const BUBBLE_LEFT_PADDING = 4;
- const BUBBLE_RIGHT_PADDING = 4;
+ const BUBBLE_LEFT_PADDING = 8;
+ const BUBBLE_RIGHT_PADDING = 8;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->